### ISSUE TITLE
MDN Layer in Keras

### ISSUE THREAD

**Issue by cpmpercussion** [cpmpercussion/robojam#13]
Need:

- split output tensor to mixture components
- PDF calculation and loss function
- sampling function for generating performances (to do).

**Comment by cpmpercussion** [comment]
This exists now at cpmpercussion/keras-mdn-layer
